File: 2 Revalida en C++/5 Funciones/9.cpp
///funcion para hallar la raiz cubica de un numero aplicando la  ley de newton
#include <bits/stdc++.h>
using namespace std;
int cubic(int a)
{

    int x1=1;
    int x;
    do
    {
        x=x1;
        x1=(2*x+a/x*x)/3;
    }
    while((x-x1)>=false);
    return x1;
}
int main()
{
    int a;
    cout<<"ingrese a:";
    cin>>a;
    cout<<"la raiz cubica es = "<<cubic(a);
    return 0;
}

File: 1 Revalida en python/5 Funciones/9.py
#funcion para hallar la raiz cubica de un numero aplicando la  ley de newton
def cubic(a):
	x1=1
	while True: 
		x=x1
		x1=(2*x+a/x*x)/3
		if((x-x1)>=False):
			break 
	return x1
a=int(input("ingrese un numero:"))
print cubic(a)